frontend/javascripts/oxalis/controller/combinations/skeletontracing_plane_controller.js

```diff
@@ -149,25 +149,40 @@ export function moveNode(dx: number, dy: number) {
 function otherNodeOfEdge(edge: Edge, nodeId: number): number {
   return edge.source === nodeId ? edge.target : edge.source;
 }
-function getSubsequentNodeFromTree(tree: Tree, node: Node): number {
-  const nodes = tree.edges.getEdgesForNode(node.id).map(edge => otherNodeOfEdge(edge, node.id));
-  const next = Math.max(node.id, ...nodes);
+function getSubsequentNodeFromTree(tree: Tree, node: Node, excludedId: ?number): number {
+  const nodes = tree.edges
+    .getEdgesForNode(node.id)
+    .map(edge => otherNodeOfEdge(edge, node.id))
+    .filter(id => id !== excludedId);
+  const next = nodes.length ? Math.max(...nodes) : node.id;
   return next;
 }
-function getPrecedingNodeFromTree(tree: Tree, node: Node): number {
-  const nodes = tree.edges.getEdgesForNode(node.id).map(edge => otherNodeOfEdge(edge, node.id));
-  const prev = Math.min(node.id, ...nodes);
+function getPrecedingNodeFromTree(tree: Tree, node: Node, excludedId: ?number): number {
+  const nodes = tree.edges
+    .getEdgesForNode(node.id)
+    .map(edge => otherNodeOfEdge(edge, node.id))
+    .filter(id => id !== excludedId);
+  const prev = nodes.length ? Math.min(...nodes) : node.id;
   return prev;
 }
 
 function toSubsequentNode(): void {
   const tracing = enforceSkeletonTracing(Store.getState().tracing);
   const { navigationList, activeNodeId, activeTreeId } = tracing;
+  let isValidList = true;
+  if (activeNodeId == null || activeNodeId === undefined) return;
```

Comment:
```suggestion
  if (activeNodeId == null) return;
```
The `== null` check checks for `null` and `undefined`.